src/locales/en/fields.yml

```diff
@@ -71,6 +81,9 @@ postcode:
   hint: "Enter the postcode in the address in section 8 of your licence"
   validation:
     default: "Enter the postcode as it appears on your driving licence"
+    maxlength: "Your postcode should be between 5 and 7 characters"
```

Comment:
Do we not need a min and max validation on postcode to trigger these

Reply:
Added.